### PR TITLE
fix: Correct static cohort import description

### PR DESCRIPTION
**Pull request by joethreepwood**
## Problem

The description for uploading a static CSV is unclear. 

It currently reads as if the single column of distinct IDs is the _minimum_ requirement -- e.g., that you should upload the distinct id column and anything extra is either ignored, or skipped. The import requires only that, but more is fine.

In reality, the actual requirement is that you upload _just_ a single column of distinct IDs. Anything else will break the import. 

This is especially confusing when you export from a dynamic cohort and receive a file multiple distinct_id columns, as well as a standard ID column.

## Changes

This PR just tweaks the wording to make it a bit clearer.

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Comment by neilkakkar** [comment]
Makes sense! Did you `git pull origin master` before making the PR? 🤔 - sounds like it's a bit out of date

File: frontend/src/scenes/cohorts/CohortEdit.tsx [hunk 1]
```diff
@@ -134,9 +134,9 @@ export function CohortEdit({ id }: CohortLogicProps): JSX.Element {
                             {({ onChange }) => (
                                 <>
                                     <span>
-                                        Upload a CSV file to add users to your cohort. The CSV file requires a
-                                        single column of the user’s distinct IDs. The very first row (the header) will
-                                        be skipped during import.
+                                        Upload a CSV file to add users to your cohort. The CSV file requires a single
+                                        column of the user's distinct IDs. The very first row (the header) will be
```
Comment (joethreepwood) [inline frontend/src/scenes/cohorts/CohortEdit.tsx:138]:
```suggestion
                                        column of distinct IDs. The very first row (the header) will be
```

**Comment by joethreepwood** [comment]
This is just waiting on a quick review when you have a minute @neilkakkar

**Comment by neilkakkar** [comment]
We updated this already

**Pull request closed without merging**